config/webpack.config.js

```diff
@@ -0,0 +1,74 @@
+const path = require('path');
+const BundleAnalyzerPlugin = require('webpack-bundle-analyzer')
+.BundleAnalyzerPlugin;
+const plugins = require('./webpack.plugins.js');
+
+const commonConfig = ({
+    publicPath
+}) => ({
+    entry: path.resolve(__dirname, '../src/js/chrome.js'),
+    output: {
+        path: path.resolve(__dirname, '../build/js'),
+        filename: 'chrome.[chunkhash].js',
+        publicPath,
+        chunkFilename: '[name].[chunkhash].js',
+        jsonpFunction: 'wpJsonpChromeInstance'
+    },
+    externals: {
```

Comment:
Externals won't work as these are not externals but rhat pulled from different app. I think that `resolve.alias` is more suitable for this
```JS
resolve: {
        alias: {
            PFReactTable: path.resolve(__dirname, './patternfly-table-externals.js'),
            '@patternfly/react-table': path.resolve(__dirname, './patternfly-table-externals.js'),
            customReact: path.resolve(__dirname, './react-external.js'),
            reactRedux: path.resolve(__dirname, './react-redux-external.js'),
            'react-router-dom': path.resolve(__dirname, './react-router-dom-externals.js'),
            graphql: path.resolve(__dirname, './graphql-externals.js')
        }
    }
```